src/main/resources/errorMessages_en.properties

```diff
@@ -35,4 +35,5 @@ AWGMENT-RUNTIME-FORM-1035=Filter should be given along with sort-by sort-order a
 AWGMENT-RUNTIME-FORM-1036=Unable to delete formData record with given formDataId and formId in elastic db
 AWGMENT-RUNTIME-FORM-1037=Regex condition failed for the given field: {0}
 AWGMENT-RUNTIME-FORM-1039=Please remove id from your payload since this is a new collection id should not be given
+AWGMENT-RUNTIME-FORM-1041=Duplicate key has been provided
```

Comment:
Please use the text-
"A duplicate value has been provided for unique field"

Reply:
I have provided the same message as you have given